packages/core/src/table/table.scss

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../styles/tokens/generated/index';
+
+:root {
+  --cds-alias-object-color-flat: #{$cds-global-color-gray-0};
+  --cds-alias-object-border-radius-50: 0.1rem; // 2px, existing radius on table
+}
+
+[cds-theme~='dark'] {
+  --cds-alias-object-color-flat: #{$cds-global-color-construction-1000};
+}
+
+$cds-alias-table-color-background: var(--cds-alias-object-color-flat);
+$cds-alias-object-border-radius-50: var(--cds-alias-object-border-radius-50);
+
+// Need to get typography colors correct
+
+// Styles and default bordering
+[cds-table] {
+  background-color: $cds-alias-table-color-background;
+  border-collapse: separate;
+  border-spacing: 0;
+  color: #{$cds-global-color-gray-700};
+  margin: 0;
+  max-width: 100%;
+  width: 100%;
+
+  td,
+  th {
+    line-height: #{$cds-global-typography-font-size-4};
+    padding: #{$cds-global-typography-font-size-1} #{$cds-global-typography-font-size-2};
+    text-align: center;
+    vertical-align: top;
+  }
+
+  th {
+    color: $cds-alias-status-neutral;
+    background-color: $cds-alias-status-neutral-tint;
```

Comment:
For the headings/highlight background you could use `$cds-alias-object-interaction-background-selected`. Its brighter than the original but aligns with what the new accordion has. It also gets you the dark theme switch automatically. https://clarity.design/storybook/core/?path=/story/foundation-interaction-styles--page